src/main.rs

```diff
@@ -159,6 +160,11 @@ fn main() -> Result<(), failure::Error> {
 
             app.spotify = Some(spotify);
 
+            app.clipboard_context =
```

Comment:
Could/should the `clipboard_context` be initialised in `app.rs`? Or is it easier to handle the error from `main`?

Reply:
Problem is that currently app initialization cannot fail. If we make it failable we will have to adjust every test. That would touch almost every file, so I refrained from doing so